apis/swagger.yml

```diff
@@ -2630,6 +2624,12 @@ definitions:
         items:
           type: "string"
           example: "c 13:* rwm"
+      DiskQuota:
```

Comment:
no, we just need to support diskquota update feature in update interface, rather than upgrade interface. @houstar 

I am afraid that you do not need to change this part in swagger.

Reply:
This patch moved DiskQuota to resource field as the originally swagger.yaml does. See this commit 91405fdaf2b5aaec561a5f2eae7ee89a9fc86420 reverted by @fuweid . This refactor not changed the pouch update interface (the pouch update command unittest works fine) but also enabled the swagger.yaml compatible with docker api.